src/elements/content-sidebar/activity-feed/common/activity-status/ActivityStatus.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import LabelPill, { LabelPillStatus } from '../../../../../components/label-pill';
+import { COMMENT_STATUS_RESOLVED } from '../../../../../constants';
+import messages from './messages';
+import { FeedItemStatus } from '../../../../../common/types/feed';
+import './ActivityStatus.scss';
+
+export interface ActivityStatusProps {
+    status?: FeedItemStatus;
+}
+
+const ActivityStatus = ({ status }: ActivityStatusProps): React.ReactElement | null => {
```

Comment:
You don't need to explicitly add `React.ReactElement | null`. TypeScript can infer the type.